scalafix-core/shared/src/main/scala/scalafix/rule/Rule.scala

```diff
@@ -107,11 +108,15 @@ abstract class Rule(ruleName: RuleName) { self =>
   // The challenge is the following:
   // - a.andThen(b) needs to work for mixing semantic + syntactic rules.
   // - applied/appliedDiff should work without passing in SemanticdbIndex explicitly
+  final def description: Option[String] = ruleDescription
```

Comment:
This method here should not be final, I think it makes sense to define it like
```scala
def description: String = ""
```
Rules that want to provide a description can `override def description = "..."`

Reply:
I will move the class parameter in an inner method in my next commit. Thanks!